src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(new File(fromFileName)));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(new File(toFileName)))) {
```

Comment:
create and try-catch reader in reading  private method, and writer - in writing private method